packages/web-components/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@microsoft/fast-components-msft",
+  "name": "@fluentui/web-components",
   "description": "A library of Fluent Web Components",
   "sideEffects": false,
-  "version": "1.6.0",
+  "version": "0.0.1",
```

Comment:
We should start this as `0.1.0` or we'll have some issues representing breaking vs. non-breaking changes.

Reply:
@dzearing I'm making an assumption here on how beachball works, but perhaps you or @ecraig12345 can confirm for me so I can resolve this :). 

I reset this to 0.0.1 and targeted my change request as a minor version change. I'm assuming that should update the package.json here to be 0.1.0 - is that correct?

Reply:
yup!